lib/suspenders/generators/testing_generator.rb

```diff
@@ -3,7 +3,6 @@
 module Suspenders
   class TestingGenerator < Generators::Base
     def add_testing_gems
-      gem "spring-commands-rspec", group: :development
```

Comment:
As someone who is anti-Spring: yeah! Go away, Spring!

But don't Spring users actually want this gem?

Reply:
I think so. Do you think we should detect if the app is using spring and if so add this line?

I see how that could be useful, but it would only be useful with `rails generate suspenders:testing`. If we generate a new app with the `suspenders` command (the classic approach), it's not that useful.

Reply:
@mike-burns Because with this PR, the `suspenders` command will ignore Spring and will generate an application *without* spring.

However, from a pure generator standpoint it makes sense! We could detect if the user is using spring, and if so add `spring-commands-rspec` when generating `suspenders:testing`. Do you think that makes sense?